content/articles/verifying-domain-resolution.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Verifying Domain Resolution
+excerpt: This article explains the different ways you can verify that a domain is resolving.
+categories:
+- Domains
+---
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+# Verifying Domain Resolution
+
+Certain changes to your domain require that its resolution be properly configured. To ensure this, we run checks to verify that your domain is resolving correctly before you make any changes. If we detect anything that might prevent a change from taking effect, a notification will appear listing the failed verifications and the actions you need to take to get your domain resolving.
+
+![screenshot: the domain resolution notificatoin](/files/verifying-domain-resolution-notification.png)
+
+## Domain is not registered
+
+Your domain needs to be registered for your changes to take effect. You can register the domain at DNSimple by clicking **Register domain** in the notification. Refer to our [guide](/articles/registering-domain/) on domain registration for more information.
+
+## The DNS zone is inactive
+
+Your domain's DNS zone needs to be active for the domain to resolve and for your changes to take effect. You can activate the DNS zone for your domain by clicking **Activate zone** in the notification. Refer to our [guide](/articles/dns-hosting/#activating-a-dns-zone) on activating a DNS zone for more information.
+
+## Domain is delegated elsewhere
+
+Your domain is not currently resolving using our name servers. Your domain needs to be delegated to [DNSimple's name servers](/articles/dnsimple-nameservers/) for your changes to take effect. Otherwise, you will need to make the changes through your current DNS provider. If your domain is registered at DNSimple, refer to our [guide](/articles/delegating-dnsimple-registered/) to learn how to delegate it to our name servers. If your domain is registered at another registrar, we have another [guide](/articles/delegating-dnsimple-hosted/) for that.
```

Comment:
```suggestion
Your domain is not currently resolving using DNSimple name servers, so the DNS records in your DNSimple account will not have any effect. Please make the the changes through your current DNS provider. 

If you would like your domain to resolve with the DNS records in your DNSimple account, your domain needs to be delegated to [DNSimple's name servers](/articles/dnsimple-nameservers/). If your domain is registered at DNSimple, refer to our [guide](/articles/delegating-dnsimple-registered/) to learn how to delegate it to our name servers. If your domain is registered at another registrar, we have another [guide](/articles/delegating-dnsimple-hosted/) for that.
```

Reply:
Your domain is not currently resolving using DNSimple name servers, so the DNS records in your DNSimple account will not have any effect. Please make the changes through your current DNS provider. 

If you want your domain to resolve with the DNS records in your DNSimple account, your domain needs to be delegated to [DNSimple's name servers](/articles/dnsimple-nameservers/). If your domain is registered at DNSimple, refer to our [guide](/articles/delegating-dnsimple-registered/) to learn how to delegate it to our name servers. If your domain is registered at another registrar, we have another [guide](/articles/delegating-dnsimple-hosted/) for that.

Reply:
```suggestion
Your domain is not currently resolving using DNSimple name servers, so the DNS records in your DNSimple account will not have any effect. Please make the changes through your current DNS provider. 

If you want your domain to resolve with the DNS records in your DNSimple account, your domain needs to be delegated to [DNSimple's name servers](/articles/dnsimple-nameservers/). If your domain is registered at DNSimple, refer to our [guide](/articles/delegating-dnsimple-registered/) to learn how to delegate it to our name servers. If your domain is registered at another registrar, we have another [guide](/articles/delegating-dnsimple-hosted/) for that.
```